File: beginner/__2000__/2813/2813.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
     
     int n;
     cin >> n;

    int guardachuvaCasa = 0;
    int guardachuvatrabalho = 0;
    int comprasCasa = 0;
    int comprasTrabalho = 0;

    string ida, volta;

     for(int i = 0; i < n; i++){
        cin >> ida >> volta;

        if(ida == "chuva"){
            guardachuvaCasa > 0 ? guardachuvaCasa-- : comprasCasa++;
            guardachuvatrabalho++;
        }

        if(volta == "chuva"){
            guardachuvatrabalho > 0 ? guardachuvatrabalho-- : comprasTrabalho++;
            guardachuvaCasa++;
        }
     }

     cout << to_string(comprasCasa) + " " + to_string(comprasTrabalho) << endl;
     return 0;
}

File: beginner/__2000__/2813/2813.ts
const fs = require("fs")
const input = fs.readFileSync("/dev/stdin", "utf8").split('\n')
const n = Number(input.shift())

let guardachuvaCasa = 0
let guardachuvaTrabalho = 0
let comprasCasa = 0
let comprasTrabalho = 0

for(let i = 0; i < n; i++){
    const [ida, volta] = String(input.shift()).split(' ')

    if(ida == "chuva"){
        guardachuvaCasa > 0 ? guardachuvaCasa-- : comprasCasa++
        guardachuvaTrabalho++
    }

    if(volta == "chuva"){
        guardachuvaTrabalho > 0 ? guardachuvaTrabalho-- : comprasTrabalho++
        guardachuvaCasa++
    }
}

console.log(`${comprasCasa} ${comprasTrabalho}`)